### PR TITLE
Initial module docstring for python _pb2_grpc

### PR DESCRIPTION
**Pull request by staticdev**
Relates to #22953 

Language: python

@yashykt

### CODE REVIEW

**Comment by staticdev** [comment]
Pair pull request for _pb2 https://github.com/protocolbuffers/protobuf/pull/7528

File: src/compiler/python_generator.cc [hunk 1]
```diff
@@ -777,7 +777,8 @@ pair<bool, grpc::string> PrivateGenerator::GetGrpcServices() {
     if (generate_in_pb2_grpc) {
       out->Print(
           "# Generated by the gRPC Python protocol compiler plugin. "
-          "DO NOT EDIT!\n");
+          "DO NOT EDIT!\n\"\"\""
+          "Protocol buffer generated client and server classes.\"\"\"\n");
```
Comment (gnossen) [inline src/compiler/python_generator.cc:781]: The current wording could be interpreted as "Classes generated *by* protocol buffers." Perhaps "Client and server classes corresponding to protobuf-defined services."?

Comment (staticdev) [reply src/compiler/python_generator.cc:781]: @gnossen I got inspiration from [here](https://grpc.io/docs/tutorials/basic/python/) but I think your phrasing is better. Can you please suggest an equivalent in the twin PR [here](https://github.com/protocolbuffers/protobuf/pull/7528)? I will change both then.

Comment (gnossen) [reply src/compiler/python_generator.cc:781]: As for your other PR, things are a bit trickier on the `_pb2.py` side. The contents of those files can't be summed up as crisply as "classes". Maybe just "Generated protocol buffer code."

Comment (staticdev) [reply src/compiler/python_generator.cc]: Changed it to the suggested wording @gnossen. Thanks.

**Comment by gnossen** [comment]
@staticdev Can you format the code with `tools/distrib/clang_format_code.sh`? This PR is currently failing sanity checks.

**Comment by staticdev** [comment]
@gnossen did it.

**Comment by staticdev** [comment]
@yashykt can I get this merged? protobuf PR was already merged.

**Pull request merged**